app/views/users/_profile_nav.html.erb

```diff
@@ -11,6 +11,10 @@
         <%= user.own_photos.count %>
       </span>
     <% end %>
+    <%= link_to 'Posts', user_path(@user.username), class: 'nav-link' %>
+  </li>
+  <li class="nav-item">
+    <%= link_to 'Posts', user_path(@user.username), class: 'nav-link' %>
```

Comment:
This is adding additional tabs

Reply:
<img width="593" alt="image" src="https://github.com/ciscovlahakis/industrial-auth-1/assets/100433760/3e81b0dd-1cdb-4d74-a8c5-ff8ff62646c1">